grails-app/services/au/org/ala/merit/ActivityService.groovy

```diff
@@ -131,6 +131,10 @@ class ActivityService {
         updatePublicationStatus(activityIds, 'unpublished')
     }
 
+    def cancelActivitiesForPublication(activityIds) {
+        updatePublicationStatus(activityIds, 'cancelled')
```

Comment:
Probably should use the constant for 'cancelled'

Reply:
I've committed the change for this.